src/Entities/Tetrahedrons.jl

```diff
@@ -89,57 +96,55 @@ function _jacobian_mat(tetrahedron_coords_matrix::AbstractMatrix, derivatives::A
     tetrahedron_coords_matrix * derivatives'
 end
 
-"Computes volume element of a tetrahedron given J = det(𝔽)."
+"Computes volume element of a tetrahedron given J = det(F)."
 function _volume(jacobian_mat::AbstractMatrix)
     volume = det(jacobian_mat) / 6.0
     @assert volume > 0 throw(ArgumentError("Element with negative volume, check connectivity."))
     volume
 end
 
-function _B_mat(deriv::AbstractMatrix, 𝔽::AbstractMatrix)
+function _B_mat(deriv::AbstractMatrix, F::AbstractMatrix)
     B = zeros(6, 12)
 
-    B[1:3, :] = [diagm(deriv[:, 1]) * 𝔽' diagm(deriv[:, 2]) * 𝔽' diagm(deriv[:, 3]) * 𝔽' diagm(deriv[:,
+    B[1:3, :] = [diagm(deriv[:, 1]) * F' diagm(deriv[:, 2]) * F' diagm(deriv[:, 3]) * F' diagm(deriv[:,
                                                                                                      4]) *
-                                                                                         𝔽']
+                                                                                         F']
 
     for k in 1:4
-        B[4:6, (k - 1) * 3 .+ (1:3)] = [deriv[2, k] * 𝔽[:, 3]' + deriv[3, k] * 𝔽[:, 2]'
-                                        deriv[1, k] * 𝔽[:, 3]' + deriv[3, k] * 𝔽[:, 1]'
-                                        deriv[1, k] * 𝔽[:, 2]' + deriv[2, k] * 𝔽[:, 1]']
+        B[4:6, (k - 1) * 3 .+ (1:3)] = [deriv[2, k] * F[:, 3]' + deriv[3, k] * F[:, 2]'
```

Comment:
It seems like unrolling this loop would be preferable (since then the rhs does not introduce an intermediate allocation), but can be done separately.

Reply:
Do you mean this:

```julia
# 1st iteration
B[4:6, (0) * 3 .+ (1:3)] = [deriv[2, 1] * F[:, 3]' + deriv[3, 1] * F[:, 2]'
                            deriv[1, 1] * F[:, 3]' + deriv[3, 1] * F[:, 1]'
                            deriv[1, 1] * F[:, 2]' + deriv[2, 1] * F[:, 1]']

# 2nd iteration
B[4:6, (1) * 3 .+ (1:3)] = [deriv[2, 2] * F[:, 3]' + deriv[3, 2] * F[:, 2]'
                            deriv[1, 2] * F[:, 3]' + deriv[3, 2] * F[:, 1]'
                            deriv[1, 2] * F[:, 2]' + deriv[2, 2] * F[:, 1]']

# 3rd iteration
B[4:6, (2) * 3 .+ (1:3)] = [deriv[2, 3] * F[:, 3]' + deriv[3, 3] * F[:, 2]'
                            deriv[1, 3] * F[:, 3]' + deriv[3, 3] * F[:, 1]'
                            deriv[1, 3] * F[:, 2]' + deriv[2, 3] * F[:, 1]']

# 4th iteration
B[4:6, (3) * 3 .+ (1:3)] = [deriv[2, 4] * F[:, 3]' + deriv[3, 4] * F[:, 2]'
                            deriv[1, 4] * F[:, 3]' + deriv[3, 4] * F[:, 1]'
                            deriv[1, 4] * F[:, 2]' + deriv[2, 4] * F[:, 1]']
                            ```

Reply:
Can address this and the other comment as follow up.